examples/bpk-component-map/examples.js

```diff
@@ -305,6 +374,13 @@ const MultipleMapsExample = () => (
   </>
 );
 
+const VisualTestExample = () => (
```

Comment:
You also need to add some cases with icons 
<img width="253" alt="image" src="https://github.com/Skyscanner/backpack/assets/29422130/723e5e20-4ac0-4c75-bbb5-0da1fbe5e42b">

Reply:
added it. thanks